Documentation/tpv34.rst

```diff
@@ -0,0 +1,168 @@
+
+Geometry
+~~~~~~~~
+
+The model volume is a half-space. The fault is a vertical,
+planar, strike-slip fault. The fault reaches the Earth’s surface. Rupture is
+allowed within a rectangular area measuring 30000 m along-strike and
+15000 m down-dip.
+
+There is a circular nucleation zone on the fault surface. The hypocenter is 
+located 15 km from the left edge of the fault, at a depth of 7.5 km.
+
+.. figure:: LatexFigures/tpv34.png
+   :alt: Diagram of TPV34.
+   :width: 15.00000cm
+
+   Diagram of TPV34. 
+
+The geometry is generated with `Gmsh <https://gmsh.info/>`_. All the files 
+that are needed for the simulation are provided at 
+https://github.com/SeisSol/Examples/tree/master/tpv34.
+
+.. figure:: LatexFigures/tpv34_gmshhalf.png
+   :alt: Diagram of TPV34 vertical fault in Gmsh.
+   :width: 15.00000cm
+
+   Diagram of TPV34 vertical fault in Gmsh.
+
+Material
+~~~~~~~~
+
+To obtain the velocity structure for TPV34, we need to install and run 
+the SCEC Community Velocity Model `CVM-H <https://strike.scec.org/scecpedia/CVM-H>`_. 
+For TPV34, we are using CVM-H version 15.1.0 and we use 
+`ASAGI <https://github.com/TUM-I5/ASAGI>`_ to map related material properties 
+for a SeisSol simulation. Detailed explanations are provided at 
+https://github.com/SeisSol/Examples/blob/master/tpv34/generate_ASAGI_file.sh
+
+.. code-block:: YAML
+
+  [rho, mu, lambda]: !IdentityMap
+    components:
+      # apply a finer CVM-H data inside the refinement zone
+      - !AxisAlignedCuboidalDomainFilter
+        limits:
+          x: [-25000.0, 25000.0]
+          y: [-25000.0, 25000.0]
+          z: [-25000.0, 0.0]
+        components: !ASAGI
+          file: tpv34_rhomulambda-inner.nc
+          parameters: [rho, mu, lambda]
+          var: data
+      # apply a coarser CVM-H data outside the refinement zone
+      - !ASAGI
+          file: tpv34_rhomulambda-outer.nc
+          parameters: [rho, mu, lambda]
+          var: data
+
+
+Parameters
+~~~~~~~~~~
+
+TPV34 uses a linear slip weakening law on the fault. The parameters are
+listed in Table below.
+
++-------------+--------------------------------+------------+--------+
+| Parameter   | inside the nucleation zone     | Value      | Unit   |
++=============+================================+============+========+
+| mu\_s       | static friction coefficient    | 0.58       |        |
++-------------+--------------------------------+------------+--------+
+| mu\_d       | dynamic friction coefficient   | 0.45       |        |
++-------------+--------------------------------+------------+--------+
+| d\_c        | critical distance              | 0.18       | m      |
++-------------+--------------------------------+------------+--------+
+
+The cohesion is 1.02 MPa at the earth’s surface. It is 0 MPa at depths 
+greater than 2400 m, and is linearly tapered in the uppermost 2400 m.
+
```

Comment:
Add:
The spatial dependence of the cohesion is straightforwardly mapped using `Easi <https://github.com/SeisSol/easi>`_.